app/server/api/ussd_api.py

```diff
@@ -0,0 +1,55 @@
+from flask import Blueprint, request, make_response, jsonify
+from flask.views import MethodView
+
+ussd_blueprint = Blueprint('ussd', __name__)
+
+class ProcessKenyaUssd(MethodView):
+
+    """
+        USSD Entry method
+        Method: POST
+
+        @params sessionId - USSD session ID
+        @params phoneNumber - MSISDN initiating the USSD request
+        @params serviceCode - USSD code dialled to access the service eg. *701#
+        @params text - User input. The initial input is blank
+
+        Method returns a text response to be displayed to the user's screen
+        All responses mid-session begin with CON. All responses that terminate the session begin with the word END
+    """
```

Comment:
public API so I don't put a `required_auth` right?

Reply:
that doesn't sound right. what stops a malicious actor just POSTING with a fake session_id or phone number? Does Africa's talking have some sort of field to supply auth headers?